spec/stack.js

```diff
@@ -13,10 +13,77 @@ describe('Stack', () => {
 
   context('push()', () => {
     it('pushes an element to the top of the stack.', () => {
-      const myStack = new Stack()
 
-      expect(() => myStack.push('foo'))
-        .to.alter(() => myStack.length(), { from: 0, to: 1 })
+      const stackOfMoney = new Stack()
+
+      expect(() => { stackOfMoney.push('foo')})
+        .to.alter( () => {
+          return stackOfMoney.lengthVal
```

Comment:
with the caveat that I haven't done this project, maybe you could add a test condition that the first item in stackOfMoney contains the string 'foo'